### PR TITLE
WIP, RFC: ramips: add EdgePoint R6 (as copy of EdgeRouter X SFP)

### PR DESCRIPTION
**Pull request by SvenRoederer**
Add Ubiquiti EdgePoint ER6, which is the outdoor version of the Ubiquiti EdgeRouter ERx SFP. (https://openwrt.org/toh/hwdata/ubiquiti/ubiquiti_edgepoint_r6)

This will essentially be a copy of the current erx-sfp device with just a different name, but allows programmatically to distinguish between both.

Before starting to rework the dts-files I'm looking for feedback if I should go on or we keep the current "use erx-sfp for r6" situation.

### CODE REVIEW

**Comment by adschm** [comment]
I'm fine with that, since it's an entirely different name.

File: target/linux/ramips/dts/mt7621_ubnt_edgerouter-x-sfp.dts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "mt7621_ubnt_edgerouter-x.dtsi"
+#include "mt7621_ubnt_e50-series.dtsi"
```
Comment (adschm) [inline target/linux/ramips/dts/mt7621_ubnt_edgerouter-x-sfp.dts:1]: I'd personally have omitted the "-series" here, and just called it mt7621_ubnt_e50.dtsi

**Comment by adschm** [comment]
@SvenRoederer I'd like to see this going on. If you feel otherwise or have lost interest, please close it instead.

**Comment by SvenRoederer** [comment]
I like too, atm just short on time, but it's one of the top prios on my list. So will work on this again.

**Comment by PolynomialDivision** [comment]
If it is exactly the same as EdgeRouter X SFP, we can just use:
  DEVICE_ALT0_VENDOR := Vodafone
  DEVICE_ALT0_MODEL := Easybox 803
  ?

**Comment by PolynomialDivision** [comment]
I added support now in:
https://github.com/openwrt/openwrt/pull/9278

**Comment by Ansuel** [comment]
@PolynomialDivision should we close this?

**Comment by PolynomialDivision** [comment]
Yes.

**Comment by KanjiMonster** [comment]
This was added with 2fd049f5cdeebebfc91084ea316ac1fa2b84a369.

**Pull request closed without merging**